pkl/defs.bzl

```diff
@@ -59,3 +55,6 @@ pkl_package = _pkl_package
 pkl_test = _pkl_test
 pkl_test_suite = _pkl_test_suite
 pkl_toolchain = _pkl_toolchain
+
+pkl_codegen_java_toolchain = _pkl_codegen_java_toolchain
```

Comment:
Should the toolchains take the form `pkl_{lang}_toolchain` or `pkl_codegen_{lang}_toolchain` ?

Reply:
I'm fine either way; one is explicit; the other is shorter and slightly more consistent with the new naming convention.

Leaning towards the latter since an extra-explicit name is not paramount in that context IMO.